website/src/pages/index.astro

```diff
@@ -1,88 +1,134 @@
 ---
 import BaseLayout from '../layouts/base/BaseLayout.astro';
 import DataInfo from '../layouts/base/footer/DataInfo.astro';
+import { BrandName } from '../layouts/base/header/Brand';
 import { organismConfig } from '../types/Organism';
 import { dataOrigins } from '../types/dataOrigins';
 import { Page } from '../types/pages';
 import { ServerSide } from '../views/serverSideRouting';
+
+/* eslint-disable @typescript-eslint/naming-convention */
+const iconMapping: { [key: string]: string } = {
+    'Sequencing efforts': 'mdi--test-tube',
+    'Single variant': 'mdi--magnify',
+    'Compare side-by-side': 'mdi--compare-horizontal',
+    'Browse data': 'mdi--database-outline',
+};
+/* eslint-enable */
 ---
 
 <style>
+    a {
+        @apply text-slate-500 hover:underline;
+    }
+
+    .items a {
+        @apply text-black hover:underline hover:decoration-4;
+    }
     h1 {
-        @apply mb-4 text-center text-3xl;
+        @apply text-center font-light;
     }
 
-    .description a {
-        @apply underline;
+    @media (max-width: 768px) {
+        .largerLogo img {
+            display: none;
+        }
+
+        .largerLogo div {
+            @apply mx-auto;
+        }
+        .largerLogo h1 {
+            @apply text-center text-6xl;
+        }
+
+        .largerLogo h2 {
+            @apply text-center text-2xl;
+        }
+
+        .infoSection h1 {
+            @apply text-2xl;
+        }
```

Comment:
I would prefer to inline those (instead of mixing inline CSS classes and this "global" stylesheet here)

Reply:
done :-)